lexer/theme-parser.y

```diff
@@ -306,8 +304,6 @@ static ThemeColor hwb_to_rgb ( double h, double w, double b )
 %type <ival>           t_property_highlight_styles
 %type <ival>           t_property_highlight_style
 %type <ival>           t_property_line_style
-%type <list>           t_property_element_list
-%type <list>           t_property_element_list_optional
 %type <list>           t_property_element_set
 %type <list>           t_property_element_set_optional
```

Comment:
Hmm, shouldn’t be `t_property_element_set` removed instead of `t_property_element_list`?

Reply:
renaming thing,  forgot the 2nd one. fixed now.